bootstrap/src/routes/Welcome.js

```diff
@@ -124,6 +85,45 @@ const Welcome = () => {
                         </li>
                     </ol>
                 </li>
+                <li>
+                    <p>
+                        In the root of the boilerplate, run <Code>yarn refresh-ldm</Code>, the terminal then
+                        will prompt you to enter <Code>Username/Password</Code> and <Code>projectId</Code>
+                        selection.
+                        <br />
+                        After that, the script will create a file <Code>/src/ldm/full.ts</Code> which will
+                        contain all <Code>LDM</Code> objects, metrics and insights that are in the selected
+                        <Code>projectId</Code>.
+                        <br />
+                        You will be able to utilize those generated objects. To read more about generating LDM
+                        objects, refer to <Code>npm run refresh-ldm</Code> section in <Code>README.md</Code>
+                        file.
+                    </p>
+                </li>
+                <li>
+                    In <Code>Home.js</Code>, replace <Code>Place your content here</Code> with actual code.
+                    <p>
+                        For example, <Code> {`<InsightView insight={Ldm.Insights.Headline}/>`}</Code>. We
```

Comment:
This will not necessarily work. The user might not have any saved insights. Why not keep the deleted mentions of the builder?

Reply:
The ticket mentions to not include any labs tooling in the inline documentation. https://jira.intgdc.com/browse/RAIL-2325

Reply:
Oh, ok :)